lib/Core/Inference/SymbolContextResolver.php

```diff
@@ -677,8 +679,24 @@ private function _infoFromMemberAccess(Frame $frame, Type $classType, Node $node
         );
 
         // if the classType is a call expression, then this is a method call
+        /** @var SymbolContext $info */
         $info = $this->memberTypeResolver->{$memberType . 'Type'}($classType, $information, $memberName);
 
+        if ('property' === $memberType) {
```

Comment:
Could extract a constant here for these magic strings (which already exist I know) e.g. `self::MEMBER_TYPE_PROPERTY`

Reply:
I preferred to reuse the constants from `Symbol`, there is an obvious and natural coupling between the two classes so it made sense to me.
Let me know if you disagree otherwise we can resolve it.